sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -536,30 +536,35 @@ def extract_output(self, accumulator):
     return accumulator
 
 
-def curry_combine_fn(fn, args, kwargs):
-  if not args and not kwargs:
-    return fn
+class _CurriedFn(core.CombineFn):
```

Comment:
This may be a performance regression.

Reply:
Given that we're calling fn(..., *args, **kwargs) anyways, this shouldn't matter. Please ignore.